<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Starter APIs" tests="4" time="14.382">
  <testsuite name="Create Employee" id="7c4c6313-dbc0-49f4-9ec6-b92041f6b41d" timestamp="2024-11-30T01:36:06.853Z" tests="1" failures="1" errors="1" time="13.939">
    <system-err><![CDATA[Iteration: 0

---
testScriptError: Unexpected token '<' at 1:1
<script>document.cookie = "humans_21909=1"; document.location.reload(true)</scr
^
---
]]></system-err>
    <testcase name="Successful response" time="13.939" classname="StarterApIs">
      <failure type="AssertionFailure" message="expected '&lt;script>document.cookie = &quot;humans_219…' to include 'success'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 968b3a18-cc72-48a2-9b6b-e8619b4aaca8.]]>
        <![CDATA[Collection name: Starter APIs.]]>
        <![CDATA[Request name: Create Employee.]]>
        <![CDATA[Test description: Successful response.]]>
        <![CDATA[Error message: expected '<script>document.cookie = "humans_219…' to include 'success'.]]>
        <![CDATA[Stacktrace: AssertionError: expected '<script>document.cookie = "humans_219…' to include 'success'
   at Object.eval sandbox-script.js:1:1).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Get All Employees" id="4ad4c0d1-49f4-4d08-b873-27988c978feb" timestamp="2024-11-30T01:36:06.853Z" tests="1" failures="1" errors="0" time="0.195">
    <testcase name="Successful response" time="0.195" classname="StarterApIs">
      <failure type="AssertionFailure" message="expected '&lt;script>document.cookie = &quot;humans_219…' to include 'success'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 968b3a18-cc72-48a2-9b6b-e8619b4aaca8.]]>
        <![CDATA[Collection name: Starter APIs.]]>
        <![CDATA[Request name: Get All Employees.]]>
        <![CDATA[Test description: Successful response.]]>
        <![CDATA[Error message: expected '<script>document.cookie = "humans_219…' to include 'success'.]]>
        <![CDATA[Stacktrace: AssertionError: expected '<script>document.cookie = "humans_219…' to include 'success'
   at Object.eval sandbox-script.js:1:2).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Put Employees" id="a9a194be-1767-4bbc-afd3-57272e5e95be" timestamp="2024-11-30T01:36:06.853Z" tests="1" failures="1" errors="0" time="0.128">
    <testcase name="Successful response" time="0.128" classname="StarterApIs">
      <failure type="AssertionFailure" message="expected '&lt;script>document.cookie = &quot;humans_219…' to include 'success'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 968b3a18-cc72-48a2-9b6b-e8619b4aaca8.]]>
        <![CDATA[Collection name: Starter APIs.]]>
        <![CDATA[Request name: Put Employees.]]>
        <![CDATA[Test description: Successful response.]]>
        <![CDATA[Error message: expected '<script>document.cookie = "humans_219…' to include 'success'.]]>
        <![CDATA[Stacktrace: AssertionError: expected '<script>document.cookie = "humans_219…' to include 'success'
   at Object.eval sandbox-script.js:1:3).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Delete Employees" id="7af832c6-2a6c-4dd2-b4bc-2cc0eeec04db" timestamp="2024-11-30T01:36:06.853Z" tests="1" failures="1" errors="0" time="0.120">
    <testcase name="Successful response" time="0.120" classname="StarterApIs">
      <failure type="AssertionFailure" message="expected '&lt;script>document.cookie = &quot;humans_219…' to include 'success'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 968b3a18-cc72-48a2-9b6b-e8619b4aaca8.]]>
        <![CDATA[Collection name: Starter APIs.]]>
        <![CDATA[Request name: Delete Employees.]]>
        <![CDATA[Test description: Successful response.]]>
        <![CDATA[Error message: expected '<script>document.cookie = "humans_219…' to include 'success'.]]>
        <![CDATA[Stacktrace: AssertionError: expected '<script>document.cookie = "humans_219…' to include 'success'
   at Object.eval sandbox-script.js:1:4).]]>
      </failure>
    </testcase>
  </testsuite>
</testsuites>